src/main/java/core/basesyntax/FigureSuplier.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSuplier {
+    private static final int MAX_RANDOM_NUMBER = 50;
+    private static final int DEFAULT_CIRCLE_RADIUS = 10;
+    private static final Color DEFAULT_CIRCLE_COLOR = Color.WHITE;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Square getRandomSquare() {
```

Comment:
do you this method outside this class? the same question for methods below. Take into account, that private methods should go after public ones